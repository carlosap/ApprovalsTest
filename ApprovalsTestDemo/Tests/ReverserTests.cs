using System.Linq;
using System.Text;
using ApprovalTests;
using ApprovalTests.Reporters;
using HangmanGame;
using NUnit.Framework;
using NUnit.Framework.Internal;
using StringUtils;

namespace Tests
{
    [TestFixture]
    public class ReverserTests
    {
        [Test]
        [UseReporter(typeof(DiffReporter))]
        public void ReverseWordTraditional()
        {
            var names = new[] {"carlos", "perez"};
            var reverser = new WordReverser();
            var reverseNames = reverser.Reverse(names).ToArray();

            //Assert.AreEqual("solrac", reverseNames[0]);
            //Assert.AreEqual("zerep", reverseNames[1]);

            //you can also add headers like so
            //Approvals.VerifyAll("header", reverseNames, x => string.Format("{0}", x));

            Approvals.VerifyAll(reverseNames, "Name", x => x.PadLeft(10));

        }
    }
}
