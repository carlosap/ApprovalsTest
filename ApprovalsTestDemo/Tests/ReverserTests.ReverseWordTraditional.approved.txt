Name[0] =     solrac
Name[1] =      zerep
